conans/cli/commands/dig.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+
+from conans.client.formatters.dig_formatter import DigFormatter
+from conans.errors import ConanException
+from conans.model.ref import ConanFileReference
+from conans.cli.command import SmartFormatter, OnceArgument, Extender, conan_command
+
+
+#  This command accepts a conan package reference as input. Could be in different forms:
+#  name/version
+#  name/version@user/channel
+#  name/version@user/channel#<recipe_revision>
+#  name/version@user/channel#<recipe_revision>:<package_id>
+#  name/version@user/channel#<recipe_revision>:<package_id>#<package_revision>
+
+@conan_command(group="Consumer commands")
+def dig(*args, **kwargs):
```

Comment:
We should be able to take the named arguments directly, aren't we?

```suggestion
def dig(conan_api, parser, *args, **kwargs):
```